dags/operators/merge_parquet_operator.py

```diff
@@ -0,0 +1,78 @@
+from airflow.models import BaseOperator
+import subprocess
+import uuid
+import os
+from airflow import configuration as conf
+from airflow.utils.decorators import apply_defaults
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
```

Comment:
I see a couple imports that are imported twice and a few that are unused (pylint is super helpful for finding these!)